### ISSUE TITLE
getPlaceDetails api response in not being parsed to onPlaceSelected 2.0.1 

### ISSUE THREAD

**Issue by bhimani07** [ErrorPro/react-google-autocomplete#96]
![Screenshot from 2021-04-22 14-08-54](https://user-images.githubusercontent.com/31835202/115765140-602c9380-a374-11eb-8447-16ea8433648e.png)

**PlaceService.GetPlaceDetails** api response is not parsed back as callback to onPlaceSelected prop after latest update

```
import React from "react"
import ApiServices from "../../../apiServices";
import Autocomplete from "react-google-autocomplete";

export default class GooglePlacesAutoComplete extends React.Component {
    onPlaceSelected(place) {
        this.props.onPlaceSelected(ApiServices.GeoCode.parsePlace(place));
    }

    render() {
        return (
            <Autocomplete className={this.props.className} onPlaceSelected={this.onPlaceSelected.bind(this)}
                          options={
                              {
                                  types: ["geocode", "establishment"]
                              }
                          }/>
        )
    }
};
```

above is the code I was using for a year now, I have moved types into options prop after recent update.

**Comment by ErrorPro** [comment]
@bhimani07 I have the same behaviour in terms of response from google. In the second version, there's no change in the interface of the `onPlaceSelected` function, it's fully backwards-compatible. You still get `getPlace()` response in the `onPlaceSelected` callback and I it's parsed. `PlaceService.GetPlaceDetails` responses header is `text`. Could you please tell me a bit more what is the issue?
![image](https://user-images.githubusercontent.com/6295152/115777808-0a78db00-a3b6-11eb-9553-eeb6ccf86749.png)

**Comment by ErrorPro** [comment]
@bhimani07 just published v2.0.3 where your issue should be fixed
